lemminx-maven/src/main/java/org/eclipse/lemminx/maven/MavenCompletionParticipant.java

```diff
@@ -110,7 +112,8 @@ public void onTagOpen(ICompletionRequest request, ICompletionResponse response)
 		
 		if ("configuration".equals(request.getParentElement().getLocalName())) {
 			MavenPluginUtils.collectPluginConfigurationParameters(request, cache, repoSession, pluginManager).stream()
-					.map(parameter -> toTag(parameter.getName(), MavenPluginUtils.getMarkupDescription(parameter), request))
+					.map(parameter -> toTag(parameter.getName(), MavenPluginUtils.getMarkupDescription(parameter), request)).filter(distinctByKey(
+							completionItem -> ((CompletionItem) completionItem).getDocumentation().getLeft()))
```

Comment:
@mickaelistria do you think there's a more elegant way to filter out duplicates by comparing a specific property?
I kind of dislike the fact I had to cast the `completionItem` to a `CompletionItem` (because the `distinctByKey` expects a function which takes in an `Object`).

Reply:
I don't have a better idea at the moment.

Reply:
> I don't have a better idea at the moment.

No worries